pkg/types/types.proto

```diff
@@ -36,6 +36,7 @@ message Interface {
 	// library, regarding each type of link. Here is a non exhaustive
 	// list: "veth", "macvtap", "vlan", "macvlan", "tap", ...
 	string type = 7;
+	uint32 rawFlags = 15;
```

Comment:
15? What happened to 8, 9, ....?
This is a protobuf message definition here, so I would expect:
```go
uint32 flags = 15;
```

Also, is it necessary to call this `rawFlags`? A simple `flags` should be enough, right?

Reply:
My first idea is that reserved other tag numbers, also numbers 1-15 require one less byte to encode than higher numbers, So I use it. If you don't mind, I would like use the number 8.

The LinkAttrs struct use the RawFlags member in kata-runtime, so maybe the rawflags has some link meanging, not just a flags

Reply:
>If you don't mind, I would like use the number 8. :-)

Yes please use 8.

>The LinkAttrs struct use the RawFlags member in kata-runtime, so maybe the rawflags has some link meaning, not just a flags?

Ok let's keep the name `RawFlags` then.

Reply:
Fixed it done.